### DISCUSSION TITLE
Flow support (re-introduce)

### DISCUSSION THREAD

**Discussion by GunnarAK** [invertase/react-native-firebase#4460] [Ideas] [answered]
This package used to have Flow support (v5) but this has been removed in v6. https://rnfirebase.io/releases/v6.0.0#main

It would be really appreciated if this was reintroduced, helping devs to maintain their code.

**Comment by mikehardy** [comment, marked as answer]
Hi there! Unfortunately, maintaining typings has a cost, and keeping multiple typing systems consistent is difficult, so we want to maintain one set of types in repo. 

We believe Typescript is the objective winner in the typings space https://www.npmtrends.com/flow-bin-vs-typescript so we chose that, and there are no current plans to support flow in repo. 

If the flow community wants to maintain flow types in https://github.com/flow-typed/flow-typed we will help with time available (and certainly won't object!) but with our very limited maintainer time available this was the best choice for the repo

I understand this is probably a disappointing answer, we hope you understand the reasoning though.

**Reply by GunnarAK** [reply]
Hi Mike, Thanks for responding so soon. It's disappointing but understandable. Thanks
